edx-platform/pearson-vue-theme/lms/templates/ccx/enrollment.html

```diff
@@ -103,19 +103,35 @@ <h2 class="hd hd-2">${_("Student List Management")}</h2>
             </tr>
           </thead>
           <tbody>
-            %for member in ccx_members:
-              <tr>
-                <td>${member.user}</td>
-                <td>${member.user.email}</td>
-                %if allow_ccx_unenroll:
-                  <td>
-                    <button type="button" class="revoke">
-                      <span class="fa fa-times-circle" aria-hidden="true"></span>${_("Revoke access")}
-                    </button>
-                  </td>
-                %endif
-              </tr>
-            %endfor
+            % if course_licensing_student_list_management:
+              %for enrollment in course_licensing_student_list_management:
+                <tr>
+                  <td>${enrollment.get('user_name')}</td>
+                  <td>${enrollment.get('user_email')}</td>
+                  %if allow_ccx_unenroll:
+                    <td>
+                      <button type="button" class="revoke">
+                        <span class="fa fa-times-circle" aria-hidden="true"></span>${_("Revoke access")}
+                      </button>
+                    </td>
+                  %endif
+                </tr>
+              %endfor
+            %else:
+              %for member in ccx_members:
+                <tr>
+                  <td>${member.user}</td>
+                  <td>${member.user.email}</td>
+                  %if allow_ccx_unenroll:
+                    <td>
+                      <button type="button" class="revoke">
+                        <span class="fa fa-times-circle" aria-hidden="true"></span>${_("Revoke access")}
+                      </button>
+                    </td>
+                  %endif
+                </tr>
+              %endfor
+            %endif
```

Comment:
1. You can reuse `ccx_members`.
2. Let default behavior and extend.
@anfbermudezme 

```suggestion
          %for member in ccx_members:
            <tr>
              <td>${member.user}</td>
              <td>${member.user.email}</td>
              %if allow_ccx_unenroll:
                <td>
                  <button type="button" class="revoke">
                    <span class="fa fa-times-circle" aria-hidden="true"></span>${_("Revoke access")}
                  </button>
                </td>
              %endif
            </tr>
          %endfor
          %for pending_enrollment in course_licensing_student_list_management:
          <tr>
            <td>${pending_enrollment.get('user_name')}</td>
            <td>${pending_enrollment.get('user_email')}</td>
            <td>
              <button type="button" class="revoke">
                <span class="fa fa-times-circle" aria-hidden="true"></span>${_("Revoke access")}
              </button>
            </td>
          </tr>
          %endfor
```